scipy/stats/stats.py

```diff
@@ -1793,11 +1800,9 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import pylab as pl
```

Comment:
This doesn't affect the code below though (except for changing ``pl`` to ``plt``).